raster3d/r3.out.vtk/writeVTKData.c

```diff
@@ -339,7 +339,7 @@ void write_vtk_data(FILE *fp, RASTER3D_Map *map, RASTER3D_Region region,
     depths = region.depths;
 
     /*the nullvalue */
-    if (!sscanf(param.null_val->answer, "%lf", &nullvalue)) {
+    if (!sscanf(param.null_val->answer, "%lf", &nullvalue) || sscanf(param.null_val->answer, "%lf", &nullvalue) == EOF ) {
```

Comment:
`param.null_val` has a default value, so it cannot be `"\0"`.

Reply:
i removed this change.